### DISCUSSION TITLE
Reference point for P2G links

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#828] [General]
The max dist between two peaks is  250,000 when calling P2G links.  Is there any bearing on TSS as frame of reference?

Thanks!

**Comment by rcorces** [comment]
> Is there any bearing on TSS as frame of reference?

I'm not sure I understand your question. can you clarify?

**Comment by sylestiel** [comment]
Is one of the peak within a defined distance from TSS of genes?

**Reply by rcorces** [reply]
peak to gene links are between a peak and a TSS. so yes

**Comment by sylestiel** [comment]
Is it 2Kb up and 100bp down (default value)?

**Reply by rcorces** [reply]
are you asking about peak2gene links or about coaccessibility?
for peak2gene links, its the distance from the peak to the single-base TSS of the gene.

**Comment by sylestiel** [comment]
The latter. Thank you!
